### ISSUE TITLE
Pretrained backbone checkpoint

### ISSUE THREAD

**Issue by PAGF188** [Czm369/MixPL#7]
Hello, I would like to know where to download the backbone pre-trained weights. That is, those that are loaded in:

https://github.com/Czm369/MixPL/blob/bb2c9aa6c32aaaf7d474f4b595f189cf00f633dc/projects/MixPL/configs/mixpl_dino-4scale_r50_fpn_90k_coco-s1-p10.py#L28

I would also like to know if you do some kind of initial teacher's burn-in to avoid the bad quality pseudo-label detections during early iterations.

**Comment by Czm369** [comment]
"resnet50": "https://download.pytorch.org/models/resnet50-0676ba61.pth"

**Comment by Czm369** [comment]
https://github.com/Czm369/MixPL/blob/bb2c9aa6c32aaaf7d474f4b595f189cf00f633dc/mmdet/models/detectors/semi_base.py#L161
